app/Http/Controllers/Auth/AuthController.php

```diff
@@ -123,14 +123,13 @@ public function create(array $data)
     public function requestOtp(Request $request)
     {
         $this->validate($request, [
-            'code' => 'required|numeric',
+            // 'code' => 'required|numeric',
             'mobile' => 'required|numeric',
         ]);
```

Comment:
why its commented out?

Reply:
in case we need it in future but now i have removed it